remind.py

```diff
@@ -395,16 +395,19 @@ def to_remind(self, vevent, label=None, priority=None, tags=None):
         elif isinstance(trigdates, str):
             remind.append(trigdates)
 
+        def abbr_tag(tag):
```

Comment:
I actually missed that in the diff :). But I think it's nicer to group method at one level, so thanks for updating it.